hw_2.js

```diff
@@ -0,0 +1,181 @@
+// #### Task 1 💻
+
+// Создайте переменные, затем сложите их и выведите результат в консоль
+//  разработчика.
+
+let x = 20;
+let y = 58;
+let z = 42;
+
+console.log(x + y + z);
+
+let sumXyz = x + y + z;
+
+console.log(sumXyz);
+
+// #### Task 2 💻
+
+// Создайте переменные:
+
+//     - количество секунд в минуте
+//     - количество минут в часу
+//     - количество часов в сутках
+//     - количество суток в году
+    
+// Посчитайте ваш возраст в секундах и поместите результат 
+// в переменную **`myAgeInSeconds`**
+
+const secondsImMinute = 60;
+const minutesInHour = 60;
+const hoursInDay = 24;
+const daysInYear = 365.25;
+
+const myAgeInSeconds = (age) => age * daysInYear * hoursInDay * minutesInHour * secondsImMinute;
+
+console.log(myAgeInSeconds(43));
+
+// #### Task 3 💻
+
+//     let count = 42
+//     let userName = '42'
+
+// Создайте две переменные. Поместите в них переменную **`count`** и превратите в строку, 
+// а **`userName`** наоборот в число. Попробуйте реализовать задачу двумя разными способами.
+
+let count = 42;
+let userName = '42';
+
+const countOne = count.toString();
+const countTwo = String(count);
+const countThree = '' + count;
+
+console.log(countOne, countTwo, countThree);
+
+const userNameOne = parseInt(userName);
+const userNameTwo = parseFloat(userName)
+const userNameThree = Number(userName);
+
+console.log(userNameOne, userNameTwo, userNameThree);
+
+// #### Task 4 💻
+// Имеется три переменные:
+
+//   let a = 1
+//   let b = 2
+//   let c = "белых медведей"
+
+// Сложите переменные так, чтобы в результате получилось выражение: 
+// **`12 белых медведей`** и результат выведите в консоль.
+
+let a = 1;
+let b = 2;
+let c = "белых медведей";
+
+const resultOneForTaskFour = String(a) + b + ' ' + c;
+const resultTwoForTaskFour = a + (b + ' ' + c);
+
+console.log(`первый вариант: ${resultOneForTaskFour}, 
+второй вариант: ${resultTwoForTaskFour},
+третий вариант:  ${a}${b} ${c}`);
+
+    // #### Task 5 💻
+
+    // Создайте переменные и поместите в них нижеприведенные слова. 
+    // Затем создайте еще одну переменную **`lengthWords`** 
+    // и посчитайте в ней длинну всех слов из списка:
+    
+    //     - доступ 
+    //     - морпех
+    //     - наледь
+    //     - попрек
+    //     - рубило
+
+const wordA = 'доступ';
+const wordB = 'морпех';
+const wordC = 'наледь';
+const wordD = 'попрек';
+const wordE = 'рубило';
+
+const lengthWords = (wordA+wordB+wordC+wordD+wordE).length;
+
+console.log(lengthWords);
+
+// #### Task 6 💻
+    
+// Создать 3 переменные разных типов и вывести в консоль для каждой из них строку следующего вида:    
+
+//     `Variable: %variable name% have type: %type variable%`
+
+const firstV = true;
+const secondV = 'bmx';
+const thirdV = 2020;
+
+console.log(`Variable: %firstV% have type: %${typeof firstV}%
+    \n Variable: %secondV% have type: %${typeof secondV}%
```

Comment:
you can also make it without \n, `` brackets work without it (\n)

Reply:
Ok